src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoTask } from '../Todo/TodoTask';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+}) => {
+  return (
+    <>
+      {todos?.map(todo => (
```

Comment:
```suggestion
      {todos.map(todo => (
```
`todos` is required, you don't need to check